android/app-example/src/androidTest/java/com/badoo/ribs/example/rib/dialog_example/DialogExampleTest.kt

```diff
@@ -0,0 +1,30 @@
+package com.badoo.ribs.example.rib.dialog_example
+
+import com.badoo.common.ribs.RibsRule
+import com.badoo.ribs.RibTestActivity
+import com.badoo.ribs.android.ActivityStarter
+import com.badoo.ribs.android.PermissionRequester
+import com.badoo.ribs.core.directory.Directory
+import com.badoo.ribs.dialog.DialogLauncher
+import com.badoo.ribs.example.app.AppRibCustomisations
+import com.badoo.ribs.example.rib.dialog_example.builder.DialogExampleBuilder
+import org.junit.Rule
+import org.junit.Test
+
+class DialogExampleTest {
+
+    @get:Rule
+    val ribsRule = RibsRule { buildRib(it) }
+
+    private fun buildRib(ribTestActivity: RibTestActivity) =
```

Comment:
Shouldn't it be built with dependencies there?

Reply:
We need the host to be ready first, as it will satisfy some of the dependencies (and since this is a UI test, we're not using mocks for them)